File: Backend/Dto/Actions/GameRestoreActionDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Backend.Dto.Actions
{
    public class GameRestoreActionDto : ActionDto
    {
        public GameRestoreActionDto()
        {
            actionName = ActionNames.gameRestore;
        }
        public GameDto game { get; set; }
        public PlayerColor color { get; set; }
        public DiceDto[] dices { get; set; }
    }
}

File: Backend/tsdto/Actions/gameRestoreActionDto.ts
/* Auto Generated */

import { ActionDto } from "./actionDto";
import { GameDto } from "./../gameDto";
import { PlayerColor } from "./../playerColor";
import { DiceDto } from "./../diceDto";

export interface GameRestoreActionDto extends ActionDto {
    game: GameDto;
    color: PlayerColor;
    dices: DiceDto[];
}
